project/src/main/java/com/google/sps/servlets/EventCreationServlet.java

```diff
@@ -23,7 +23,8 @@
 
 @WebServlet("/create-event")
 public class EventCreationServlet extends HttpServlet {
-  private static final DatabaseWrapper dbWrapper = new DatabaseWrapper("step-188-instance", "event-organizer-db");
+  private static final DatabaseWrapper dbWrapper =
```

Comment:
Please use the new style with SpannerTasks that has been merged to master.

Reply:
yup, will take the correct change in merge